src/PDO.class.php

```diff
@@ -264,7 +275,7 @@ public function insert($tableName, $params = null)
 	{
 		$keys = array_keys($params);
 		$rowCount = $this->query(
-			'INSERT INTO `' . $tableName . '` (`' . implode('`,`', $keys) . '`)
+			'INSERT INTO ' . $tableName . ' (`' . implode('`,`', $keys) . '`)

```

Comment:
2. Insert `schema.table` will show error exception.
If the program across two db.
When use `$db->insert('db2.table', para);`
```
PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db.db2.table' doesn't exist.
```
If remove \`\` I don't need to `new DB(Host, DBPort, **another_db_name**, DBUser, DBPassword)`